src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let total = 0;
+
+  function adder(...args) {
```

Comment:
The `adder` function should accept only one argument at a time. The use of rest parameters (`...args`) is not necessary and goes against the required function pattern. It should be refactored to accept a single argument.